exchangeAPI/src/handleErrors.js

```diff
@@ -0,0 +1,69 @@
+let symbolList = [];
+
+var myHeaders = new Headers();
+myHeaders.append("apikey", "K1wPo186HgThBr8LBU4xPtV0QHX2p63S");
+
+var requestOptions = {
+  method: "GET",
+  redirect: "follow",
+  headers: myHeaders,
+};
+fetch("https://api.apilayer.com/exchangerates_data/symbols", requestOptions)
+  .then((response) => response.json())
+  .then((result) => (symbolList = Object.keys(result.symbols)))
```

Comment:
I think you can use the [push](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push) method here.